Cargo.toml

```diff
@@ -33,7 +33,7 @@ default = ["presto_expressions"]
 presto_expressions = []
 
 [dependencies]
-datafusion = { version = "38"}
+datafusion = { git = "https://github.com/sdf-labs/arrow-datafusion", branch = "zhong/v38" }
```

Comment:
Don't change this

Reply:
Done: Reverted to the same entry as on main.